packages/kit/src/exports/vite/dev/index.js

```diff
@@ -187,9 +188,7 @@ export async function dev(vite, vite_config, svelte_config) {
 									query.set('inline', '');
 
 									try {
-										const mod = await loud_ssr_load_module(
-											`${url.pathname}${url.search}${url.hash}`
-										);
+										const mod = await loud_ssr_load_module(`${pathname}${url.search}${url.hash}`);
```

Comment:
by doing the `decodeURI` call outside the `if` you're doing it all the time, but really you only need to do it for CSS requests
```suggestion
										const mod = await loud_ssr_load_module(`${decodeURI(url.pathname)}${url.search}${url.hash}`);
```